packages/apps/job-launcher/server/src/modules/job/job.repository.ts

```diff
@@ -126,4 +131,110 @@ export class JobRepository extends BaseRepository<JobEntity> {
 
     return { entities, itemCount };
   }
+
+  async getAverageCompletionTime(): Promise<number> {
+    const queryBuilder = this.createQueryBuilder()
+      .select('AVG(completion_time_seconds)', 'average_completion_time_seconds')
```

Comment:
The view is not actually necessary
```suggestion
      .select('AVG(EXTRACT(EPOCH FROM (updated_at - created_at))/60)', 'average_completion_time_minutes')
```